packages/cli/src/modules/chat-hub/chat-hub.service.ts

```diff
@@ -897,7 +897,7 @@ export class ChatHubService {
 		};
 	}
 
-	private buildMessagesGraph(messages: ChatHubMessage[]) {
+	buildMessagesGraph(messages: ChatHubMessage[]) {
```

Comment:
Any particular reason to make this public?

Totally optional: now that this function doesn't contain any substantial logic, we could also refactor this with fn like
```ts
convertToDto(message: ChatHubMessage): ChatHubMessageDto;
```

Reply:
And yes, happy to refactor it a bit now anyways